tests/service/predictors/decomposable_attention_test.py

```diff
@@ -39,3 +39,41 @@ def test_uses_named_inputs(self):
         sumexps = sum(exps)
         for e, p in zip(exps, label_probs):
             assert e / sumexps == approx(p)
+
+    def test_batch_prediction(self):
```

Comment:
Not really complaining here, but this was more work than you needed.  Because the functionality is in the base class, you really only need one of these models to have the test for batch prediction.  It could even be in a `predictor_test.py` file, using an arbitrary model for the test.